### ISSUE TITLE
Clarifications on Caching

### ISSUE THREAD

**Issue by PathToSharePoint** [pnp/pnpjs#1496]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [latest]

Please specify what version(s) of SharePoint you are targeting: [Online]

### Questions
I am interested in implementing [Caching ](https://pnp.github.io/pnpjs/odata/caching/#implement-custom-caching) and I have ~~two~~ three questions:
- What are the default settings? The documentation shows how to overwrite them, but doesn't tell what they are in the first place.
- When caching and batching are combined, does caching apply to the whole batch or to each call individually? For example, if I query 3 lists in a batch, and at the next iteration I query one more list (making it 4 in total), will PnPjs reuse the cached calls?
- What are the size limits? I am thinking in particular about queries of file content.

Thank you!

**Comment by PathToSharePoint** [comment]
I found the caching defaults under [Common Configuration](https://github.com/pnp/pnpjs/blob/07d1130fdc4257a9ed62563dec85d95578145af0/docs/concepts/configuration.md#common-configuration).

As for size limits, I am assuming that "session" and "local" match the storage API terms. I tried to find the current limits, and it seems it's 10 MB (**total** storage per origin). I couldn't find a reliable source.
